net-news/sfeed/sfeed-1.2.ebuild

```diff
@@ -12,29 +12,33 @@ SRC_URI="https://codemadness.org/releases/${PN}/${P}.tar.gz"
 LICENSE="ISC"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE="ncurses"
-
-DEPEND="ncurses? ( sys-libs/ncurses )"
-RDEPEND="${DEPEND}
-	net-misc/curl
```

Comment:
curl is used by sfeed_update and shouldn't be considered an implicit system dep (unlike coreutils, glibc, and virtual/awk to a rough extend).

Reply:
curl is an optional runtime dep. should be opt-feature i think.